index.js

```diff
@@ -0,0 +1,9 @@
+const tileOnClickHandler = () => console.log('tile number');
+                                                                    
+
+const square = () => {
+    const tile = document.createElement('div');
+    tile.addEventListener( 'click', tileOnClickHandler );
+    tile.className = 'cipka';
+ 	return tile;									
```

Comment:
Indent to tile word above

Reply:
It does looks differently (correctly) in my text editor for some reason